scripts/update_python.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# Copyright 2020 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+"""Updates the python binaries that we cache store at
+http://storage.google.com/webassembly.
+
+Currently this is windows only and we realy on the system python on other
```

Comment:
Also will this still be true when we start needing python3 on mac?

Reply:
Also, it looks like this is intended to run on a unix system, since it relies on the zip command-line tool? (the other option is what we do in the [waterfall](https://github.com/WebAssembly/waterfall/blob/master/src/build.py#L601) (also [here](https://github.com/WebAssembly/waterfall/blob/master/src/build.py#L332))

Reply:
We may include a bundled version of python on mac yes, but "Currently" we only do it on windows.  

I have not particular desire to make these scripts run on windows at this point.